crates/astria-conductor/src/executor/mod.rs

```diff
@@ -222,6 +229,29 @@ impl Executor {
             return Ok(None);
         }
 
+        match self
+            .executable_block_height
+            .cmp(&(block.header.height.value() as u32))
+        {
+            std::cmp::Ordering::Less => {
```

Comment:
I think it's ok to roll this into one. The distinction between `future` and `stale` will be lost when quickly scanning the logs.

If a `if self.executable_block_height != block.header.height` { ... }` is cleaner.